app/assets/stylesheets/base/typography.scss

```diff
@@ -33,7 +33,7 @@
     @apply mb-2;
 
     &:last-of-type {
-      @apply mb-0;
+      @apply mb-2;
```

Comment:
We may be able to get rid of the `&:last-of-type { ` targeting completely; since the `@apply mb-2` is included in the higher selector